web_widget_bokeh_chart/__manifest__.py

```diff
@@ -6,13 +6,13 @@
     "category": "Hidden",
     "summary": "This widget allows to display charts using Bokeh library.",
     "author": "ForgeFlow, " "Odoo Community Association (OCA)",
-    "version": "14.0.1.0.0",
+    "version": "14.0.2.3.1",
     "maintainers": ["LoisRForgeFlow"],
```

Comment:
:(

Reply:
I think the problem might be from the change on oca_dependencies.
:cry: